python_ta/typecheck/typeshed/builtins.pyi

```diff
@@ -297,7 +297,7 @@ class str(Sequence[str]):
     def __ge__(self, x: str) -> bool: ...
 
     def __len__(self) -> int: ...
-    def __contains__(self, s: object) -> bool: ...
+    def __contains__(self, s: str) -> bool: ...
```

Comment:
Hmm this is interesting. In principle this file should be an exact copy of `https://github.com/python/typeshed/blob/master/stdlib/3/builtins.pyi`, so I'm hesitant to change it. On the other hand, it does seem like any non-string type is an error. Can you please create a `README.md` under typeshed to make a note of the changes to this file?